packages/blockchain-info-components/src/Images/Images.ts

```diff
@@ -281,6 +284,7 @@ const Images = {
   'swap-white': swapWhite,
   'terminate-card': terminateCard,
   'tier-gold': tierGold,
+  twitter,
```

Comment:
https://blockc.slack.com/archives/C02UVTY5U5C/p1653747099280809?thread_ts=1653331195.118129&cid=C02UVTY5U5C

we should be using icons